pkg/unarchive/dmg.go

```diff
@@ -109,31 +105,27 @@ func (unarchiver *dmgUnarchiver) Unarchive(ctx context.Context, fs afero.Fs, bod
 		m = io.MultiWriter(f, bar)
 	}
 
-	err = writeDmgFile(m, body, dest)
-	if err != nil {
-		return fmt.Errorf("writeDmgFile failed :%w", err)
+	if err := writeDmgFile(m, body, dest); err != nil {
+		return fmt.Errorf("write a dmg file: %w", err)
 	}
 
 	exe := exec.New()
 	tmpMountPoint := destDir + string(filepath.Separator) + "mount"
 	_, hdiutilDetach, err := exe.HdiutilAttach(ctx, dest, tmpMountPoint)
 	if err != nil {
-		return fmt.Errorf("hdiutil attach failed: %w", err)
+		return fmt.Errorf("hdiutil attach: %w", err)
 	}
 
-	err = cpDir(fs, tmpMountPoint, destDir)
-	if err != nil {
-		return fmt.Errorf("failed cpDir src:%s,dst:%s :%w", tmpMountPoint, destDir, err)
+	if err := cpDir(fs, tmpMountPoint, destDir); err != nil {
```

Comment:
nits. I'd like to make the scope of the error variable small.

Reply:
I agree! Thank you!